capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -50,17 +50,17 @@ export class WikiMediaComponent implements OnInit, OnDestroy {
       let endIndex = firstPartOfString.indexOf("<h2>", 0);
       let startIndex = firstPartOfString.indexOf('</h2>', 0);
       let middleOfString = firstPartOfString.substring(startIndex, endIndex);
-      let finalString = middleOfString.split('href="/wiki').join('target="_blank" href="https://en.wikipedia.org/wiki');
-      finalString.split('title="Edit section: History">edit').join('>');
-      return finalString;
+      let paragraphs = middleOfString.split('<p>');
```

Comment:
Does this work entirely correctly? Wouldn't that give you turn text "<p>Example</p" into "Example</p>"? A better idea might be .split(/<.*?p>/); to get just "Example"

Reply:
It does work.